src/Input.test.jsx

```diff
@@ -0,0 +1,52 @@
+import { render, fireEvent } from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input component', () => {
+  const handleChange = jest.fn();
+  const handleClick = jest.fn();
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const renderInputComponent = (value = '') =>
+    render(
+      <Input onChange={handleChange} onClick={handleClick} value={value} />
+    );
```

Comment:
```suggestion
const renderInputComponent = (value = '') => render((
    <Input
      onChange={handleChange}
      onClick={handleClick}
      value={value}
    />
  ));
```

이렇게 작성하면 props를 보기 더 편할 것 같아요